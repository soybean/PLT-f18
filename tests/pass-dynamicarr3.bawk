BEGIN {}

LOOP {}

END {
  int[] a;
  int b;
  a = [1,2,3];
  b = a[3]; # Fail: Accessing past size of array
}
