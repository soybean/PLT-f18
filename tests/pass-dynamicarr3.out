Index Out of Bounds Error.
